### DISCUSSION TITLE
Should the Homebrew docs recommend using the GH cli instead of hub?

### DISCUSSION THREAD

**Discussion by while1malloc0** [Homebrew/discussions#3005] [Everyday usage]
When I was going through the docs the other day, I noticed that the recommendation for getting PR branches of formulae involves using the `hub` CLI: https://github.com/Homebrew/brew/blob/master/docs/FAQ.md#how-do-i-get-a-formula-from-someone-elses-pull-request.

Should that recommend using https://cli.github.com instead? I know that Hub is still maintained, but that particular code snippet in the docs was last updated 5 years ago and I wasn't sure if the Homebrew maintainers prefers to use the first-party tool or doesn't feel strongly.

I'm happy to PR a docs update, just wanted to get thoughts first.

**Comment by SMillerDev** [comment]
Yeah, seems like that just hadn't been updated. If you can make a pull request I'm sure it'll be accepted.
